h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -239,6 +264,33 @@ public void resize(int len) {
   private transient double _ds[];   // Doubles, for inflating via doubles
   public transient byte[]   _ss;   // Bytes of appended strings, including trailing 0
   private transient int    _is[];   // _is[] index of strings - holds offsets into _ss[]. _is[i] == -1 means NA/sparse
+    
+  public long byteSize() {
+    long mem = 0;
+    if (_ms != null) {
+      mem += _ms.byteSize();
+    }
+    if (_missing != null) {
+      mem += _missing.size() / 64;
```

Comment:
The internal array is long[] (is that guaranteed ?), but the `size()` method is supposed to return total amount of **bits** allocated. By diving it by 64, don't we get the length of the `long[]` internal array ? In other words, don't we get the `.length` of that array effectively ? If we want bytes, shouldn't we just divide by 8 ?

Sorry if there is something I'm missing.

Reply:
good catch!

Reply:
the fact that internaly it uses long is not guarateed, all this is on a best-effort basis